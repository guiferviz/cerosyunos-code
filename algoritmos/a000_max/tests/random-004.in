28
62
1289458961 2107311068 781586634 1340411174 1281818510 288403417 1515607232 1281276641 398245406 113009436 743123733 501990590 595573037 730803418 1330327509 938154140 1782747511 407716973 1176075335 877226892 2126270827 1340516343 979905252 300766223 149816669 132661618 1392124475 1592275012 911964772 1946165535 245214747 207552209 777003866 296820161 138507026 1676172019 204533449 237934770 1051737981 1238425858 351225411 1844970699 1092289910 1892861997 1696016496 2061683132 1319991936 1415795253 1993753829 913868076 1523660485 1187061655 515428228 10955622 584382652 738450147 550472966 598954424 1894213664 1532984213 337518234 1072152965
62
1735066967 315857551 1093468516 1127979502 657500364 2073705305 803284414 482217470 1041175166 833031881 634447922 1890455639 775000664 839428453 951005822 1532855420 173733015 423374832 1259572243 2104276214 90706157 1360975518 1199777083 1112379100 955287649 2032384603 1043285254 1769796952 430291201 266332479 757855352 1391778901 1772060716 981351952 1055008209 2081232226 1135898576 2072056947 438273211 86511788 1752200622 1923376534 232585523 1395890655 1481869571 600157886 1141946942 1512103831 623971967 492534942 696310569 2131574079 558863319 189087209 1346219521 934504905 432800803 183047664 677376845 847959276 1845285473 929815511
62
330522202 1217981365 1530924563 780351505 1435908662 1165700980 219319127 1944080939 716937906 625429199 797655486 2046541959 244778608 666812657 648995512 581623845 519777483 1583657131 1238553306 2083966504 1132167290 819893713 74718473 1999743845 1828217137 1012179486 1582619366 1398898257 1392803437 1105129472 1816653716 806312986 38146666 548723393 564441998 1054049039 729485482 880793682 1369734524 128829785 2006194505 1390004385 1744128988 180429449 1520617274 810279670 226829114 1856244943 2109667415 225089900 679145159 350039167 2074241687 2000339752 541286158 272609824 382999794 859718352 139620375 2034202528 363480386 1080321910
62
1053303603 1261466865 104524877 1872062539 1387964497 1037707107 60812011 503895111 1469281544 603263979 1953386081 25027812 294382203 674110553 721471698 2048236977 1785827077 392089650 422113114 1153047371 287500038 546820773 687448370 410434485 741795236 206216334 1364967476 756801526 1453251750 1776537349 651143085 307688083 122058480 699087620 268898583 641658631 1629028214 18503719 743105098 13341222 1550927667 1227460307 245109143 459679559 907090282 2080179669 300668153 999503191 94550475 185732653 1487160201 1507236029 1878903770 499975655 1714358286 345210432 2111012224 1982536196 377630779 1306745756 1980096805 1212384984
62
1435138252 317134389 1405008642 94513317 1251623409 1237501311 234203943 1384957963 1013990962 912153522 2133157827 2112724561 1868413496 614924741 776566499 881247120 1165992262 959097109 1266236851 1602216912 664875001 681665013 1590174169 231235975 1361419869 1745907179 585807366 1267578668 848547155 725249201 84469787 1653350779 1973784148 1414898547 1675655926 450558836 1542822564 1655420708 1509128025 1831140167 684418178 463048700 1866201483 1150570152 1915743471 1798562040 950017172 1585941402 906280701 1552762535 1014942137 1458883937 1484779228 667102989 577548511 2087211764 1549508975 104141965 1163373610 2108218715 1236073336 962039815
62
1321063539 1181357073 670316237 560190654 1012721684 1483680762 1460964427 1735725193 604312526 739578580 696713192 756167102 1045375464 1713602475 142411123 1905526360 1074264957 779475777 2141711101 1389176715 273947065 874857963 176120366 209952656 400108684 1081623833 965805457 38630688 344012124 1979304600 244381628 2070166929 594097413 221383093 471640461 748559190 1518528951 862096145 1971767269 485891428 1244066956 1680041668 2126592747 1567571108 1865904466 1384731740 732170286 1576581687 747020500 1651140357 628428825 235545347 476562852 283145153 374369784 168051267 1237102348 1141164100 1471255046 189352247 148830360 453416688
62
663160104 1381788538 1224623029 1166497399 175832740 1213759130 69029162 1190527914 163069158 969961845 1188650267 1533747361 487816864 298303752 1298860316 128900044 1680378904 1909894124 1469511575 1991291869 25561043 1936941632 1516804092 1300952121 482750158 1679155848 201440332 838814668 516581397 126002078 1451150864 718528018 1408408433 1202458183 1707761940 1224703660 14176098 643448428 1404817185 121654791 258056052 1056554372 967702060 917869745 1780240602 1042562782 833989555 63181 1860032870 1331116075 1164990996 788732562 728124998 374269262 1702866004 1359597008 1138819953 27604034 1036336625 1581409176 458663465 1473696274
62
1851404491 2130269096 843409083 1425035181 1677747128 1441888096 918441019 2008669994 535201664 1615467212 301985150 1105683250 128879228 484954539 890137514 1662523796 1031548206 245767470 1684308770 1004789425 1933002199 520216353 115719556 859670543 1748381415 445907981 281994628 798170443 1875150507 238667254 1841694733 1370363549 931101100 729809182 1497382120 1984707849 1557141147 394551363 1634019367 81967312 121837344 589864755 814094324 1711042014 627679688 220604941 242187461 710404663 1942789794 1155782217 2118848674 1332668636 1910050195 479640549 800813286 1213029617 2085000083 266493145 1768669553 1602163376 1387222405 1595516213
62
261567988 1765219728 595673972 13349999 315329417 12112711 908797939 516164020 131795499 386814091 183069655 1582189455 1956482463 1508450925 51721875 1767925143 783823662 830253620 1189391029 967895441 1471406445 780413749 1517956807 1685085446 382899872 1172922201 1436248389 1047353548 1725179812 213341549 1176882048 752939194 1621675003 662201726 1560422128 1589692008 1196565871 583450082 748647122 624677924 567717235 723568794 2064608244 1770779373 1761850294 1613344249 1006547338 1823681609 2069610966 1867776758 1500285062 1471240114 239471514 1901454753 1463464649 766664436 720197544 1325194396 866825558 1091718811 649136921 386100924
62
915197020 1689110465 1672499717 993970268 217450096 799653398 166343703 1300695802 1122615062 952733906 694220055 48793577 184877962 12834695 243352603 613193876 598172759 1302119145 1497520832 226192705 558843516 293908598 795703291 82455327 281689173 1315259618 337750716 643717292 594078000 297278257 901847209 2036921917 88861851 1174937568 827271534 362012347 667800716 839924547 1184792052 1447243009 1200362709 953531607 725836930 929155558 1923727706 1168359205 807363073 52219802 951412225 1185217624 1802363265 408660053 1846996520 1514912829 1906384074 2042077592 250824315 1413794436 1854865991 694359966 534029454 428972703
62
33940190 1317693585 1211313246 1485200395 1869856387 513706249 12480 517809492 1270241260 919697477 1921994196 1032617619 1612134286 191027998 1132127185 658110704 39987209 680108978 1550671434 1641529013 1131630095 29210892 1441751727 516243995 114635534 1764458047 409355178 506139950 1631678463 449800208 2093821390 1191644208 226143775 1640379578 1648327393 1697193968 1456761245 2024281934 1601374589 1261870205 960534013 471997278 855370178 745135154 2119494938 629852187 459142834 1046083152 907143031 141538521 587033729 1763273156 914246569 237425480 663104231 399016065 1950731742 1127986485 803848560 293484942 2051002323 199368544
62
1567040103 1005899160 1431132218 1195142797 955658517 1413952380 590883725 410161169 888020218 1947498241 705349596 1012671506 1891569435 957848749 752571905 482710247 440248201 1852388257 642226447 238664208 1436719566 806840397 1877112787 403612710 1575627435 968582881 1186187318 2004380453 1427887783 794008320 1742365604 469013227 1824219539 1154366902 109646263 1752136528 409297393 1769882771 1453281923 1240851105 1287981379 75730122 1987860620 1986966282 803372007 150005303 1924379953 32024406 171140750 2002886634 465647875 1061544760 1945850263 1788080066 1601613447 821396607 593354976 1296231181 782760303 84637421 1368766370 939246889
62
1639067183 286784363 2006014754 1122583447 714789882 1727660151 934386007 1476504708 1829428998 720493500 1850725995 630122888 126492016 944775082 382333987 589467228 1210362176 1338275466 1257690808 246674067 1058510791 259384200 1753107 964029414 379632352 1117343407 1118611762 1826112679 816657806 1491282763 728855576 249890569 1226211170 468894611 1514832396 1512193562 1650859690 964093772 668663145 829425771 1425315094 816724503 1041490013 234934583 1652177554 1094327463 769658761 957716974 1792831330 167296015 1165806433 1392712539 650202088 1559848779 172149684 1160347494 537224726 1936978 1718457166 1485811886 1517874028 1385164348
62
528394302 1699513731 822089460 610612481 786831750 1614338557 1667052102 64838406 753858393 2041312901 1366544837 1246282289 2029707514 1436516909 1564470654 1675272016 516950823 630875291 1866215652 547971854 1653454251 399285012 1411212467 401169372 1570947098 722484527 1130965146 1698779561 550851852 684362079 364326553 1685637822 235277549 1375938639 1884573272 109291455 719490704 1644779677 613707526 1690620900 350729247 220440869 207118746 1356750573 899886161 2018521645 55057377 1433160283 791847550 1762730563 533552366 347271464 1143671011 97763593 855916203 655431173 1296772708 746790266 13916176 1748205603 661859810 1117960492
62
1203440019 427578534 1638917881 101952199 348925323 2076794862 1613144597 1697743682 1254864415 918839203 236770628 1509868140 926213789 235514442 1907091324 405691841 2057620233 783729868 2073239822 529024573 534766268 2058333421 201627247 1040071999 531341859 1092806360 1036495886 1541707168 1598781264 1798486938 845952925 2138091213 1111378912 1012333192 1495582094 573129990 1547150388 645229054 1302660480 1521873032 2108020963 1583538120 1687756162 1443370429 1226385704 1787331900 263099980 1170257167 1931741013 1249157052 1380824615 573850149 944894512 202156229 1522028917 1946615657 747341401 2090997933 1762776636 968021299 296857327 423297948
62
1788550058 657126345 841156070 1132059773 1863818037 1575325322 735177205 536361287 1245760683 1220338452 1813685766 982917258 630373285 1634110093 624121750 303233338 888173992 2043546809 1908405499 557182562 771585476 1914583161 1103862878 1475231815 1067313264 931835224 2104894934 1314613588 193060460 1502417492 1858907797 469969285 1626574906 926131952 531162249 110794515 1168449182 408545322 1332023845 1492053638 394439197 645103612 547779702 1790404714 678446697 1230023177 912538864 1171034272 1213806963 697124803 396825649 1002303497 1528062314 1339699715 1109573647 981421960 687514411 506466564 1160202313 1688392223 1234767242 2109695018
62
379289107 698237116 487824521 1421490346 636130141 272297810 602546706 467869587 288823049 1785929457 1944958805 1800912984 230826231 1422507960 2102339633 1406195282 1321872695 103986533 478744861 869668792 370303393 309972458 469860690 1149103038 2099098914 1997727715 744007063 1611532611 1873752103 231199816 1579562219 718260017 2017058445 462289324 1703764317 2132788483 518472011 1601509810 1901986356 2057404598 1546878887 1865076486 436051473 1290629740 1505005276 749298785 1105843507 1780098145 1130765606 305816641 1787023117 2116520772 1208036304 1784371591 1340975573 1541812177 789152996 504078053 791168658 969814349 911261377 182844606
62
2051230888 1768248128 167342031 131950398 57786249 894600213 2020420460 907187197 569803932 1204155454 555996335 1674820886 1956509798 2138750975 424922504 366413777 37519032 209508690 789394249 767867871 2053895075 166971093 793515906 710766773 1609689727 1287623327 1343268753 861628292 1626913900 631597873 592967400 2011064105 1497041799 700960408 55806362 1267985655 630754301 68622185 740296126 1861092602 1809759696 24916072 1102382077 908873998 36657133 1035829499 1376434854 1964254724 962993551 1628737492 737750404 1130908353 1739877 920225423 1301200501 1595819186 395735721 1532410246 585384500 372840578 157312300 444727805
62
193181282 1198881386 1438980912 2102541138 685105685 46504880 1752910606 1033886278 1455409143 463281418 462584392 438705150 636760124 191999468 1897116840 973032111 1131345631 100152366 1868582639 2011047276 1559591242 1914163026 401348613 282471767 1917772529 169797313 377417029 207176670 67918618 575370103 1311444061 895835658 1661528031 883159842 1199087264 1739688937 1407155165 1166639930 1012485196 746817565 555549463 124672095 621081346 1955687512 593097966 1687053051 1485012254 1694436732 1407479706 815742541 1667614959 417526431 1421371353 1108015231 612946542 743747475 2128377198 1615153737 1987418008 1626790342 1637879187 2104121002
62
716499831 1086165516 400630106 456609123 1325282607 1026045872 1132790437 1556449066 1944103552 1221985692 844372473 45378625 369726065 1442666176 1895573809 793759482 938188323 338958476 824498121 1314925733 1837729883 1734717648 140289958 242204938 905000834 2026340898 536470069 859699305 1779363356 79102430 764502874 975811201 1154342053 198079388 1725080884 1455372283 1114755751 563636583 939317015 64579680 1991946937 1777842198 1914045706 140495892 1382984767 738390932 1474928231 2004931923 406780580 1148767505 2072866278 232549993 1011026461 1440953578 1901698265 839906475 606017468 798749624 1930197757 230819894 1255250059 896570797
62
1078506724 1031695160 372206104 1038172777 1382975550 947316626 1942880023 1639255229 1348879940 1312420044 994697035 1690658225 914999189 262923847 961952405 529627194 448914892 1823549971 1342535498 1720345952 1027857337 1495912114 683297797 302541634 545776929 1880477256 1943418032 1782388999 839879953 1042486181 771963077 558806632 130533052 290461224 1925260412 2035237116 629014099 1074552068 1277713169 1660716216 1822796989 1974644776 877315059 1295647612 1276736997 1262698484 1421512903 1988416981 1738419655 1466768514 371422669 683256343 1865712969 746080889 745580718 1850602988 439665793 1704792061 1484198564 97491186 2137654538 1057399442
62
866779496 301052570 600714538 901692231 527450897 1972067049 2023731229 847068400 207645162 657488890 1632356164 1846270074 1695748239 1803988203 1032788967 1261079825 1333195447 700895656 898872369 2092038462 69475348 2003634862 1149822820 1754054985 1840377840 1226063480 1313690020 1701761858 603347076 1198263554 1886717783 179041616 697949968 659288640 2075722596 1236747057 1580618092 1564918240 2038065919 425476029 495373790 133411845 1515542177 1912112240 1489133263 1303585536 1806458966 1381452257 1096317837 1855854148 1765631045 1032092849 1402634137 97666910 1779096161 1154874528 1312913645 1800347172 1644861649 1646908777 1790028825 134259495
62
1088427796 424538227 796261655 627950073 1604587111 779111628 1160590950 689808854 752993255 1963094719 1229187698 1263128218 1062418318 444783321 20155375 151079717 686431003 609917412 533360477 1262254174 1067011332 1558115573 1413405646 200789497 657949938 1093156648 947880429 750793415 1302638421 1215589316 1935338169 243322411 298942334 996376545 921465114 1693271492 673061488 1101251448 446888099 444405160 1807711671 458548968 314560955 1039704816 1094967009 686677926 1124487578 1198629422 893678452 1372501724 812458849 53271272 1101770002 1521285299 1424826291 156199823 502198472 1438266704 2124719015 704498953 1788604551 528085988
62
1631139947 1620768838 1918433922 606456951 2065167730 1447218447 1767767826 1312055244 1902748702 50821586 318672760 1766388562 1197538210 1511015923 2077750259 610716473 1010325574 1845547121 323849728 473635366 142278818 1397364941 396962249 1806121527 1115189103 1334677997 800123069 308032935 1540398124 1156640095 678219856 812437620 1091910817 1217502121 927428641 30889425 1558498000 2039557419 185815318 1596135347 1965405885 721413738 2077714576 1901686528 1088421822 1903225052 1652419584 1518367760 824956022 129355852 1949547903 1462513730 1552656407 1225442023 1728929922 895171468 287627602 1436979296 272084344 366417407 161718361 1158437467
62
1643139172 843892785 994202338 1744936766 383462370 2058511317 96561039 1788619749 242038195 1854164765 1443907095 1497706529 1202029032 1837906437 935028423 649756506 1626712628 491399989 2057616506 137102566 1031326369 1834943969 1184254081 1569814825 1077632934 515922836 1315303020 159938262 94694129 2092357744 1751834750 1875169773 702932458 1806327508 435387784 674222815 937861493 2102919450 1940417088 323855415 744600365 1984771697 1022798481 317417952 1692761959 471961617 1166786080 1462619243 1852733664 1939170165 1507957619 1425571618 50453977 1306252551 103251923 1396289775 745245508 1001412084 280627749 1827154124 1949966799 294947401
62
1586032379 1580541748 1182628930 1362774357 953014447 1507467974 1694771757 1503658500 1164250750 866144500 184676599 1601880388 450685505 304194 1047609638 1941182682 601266170 885917591 367442674 2084813982 1405252381 1152536698 419676198 1161802861 1070906191 746682596 505887870 732532603 619331841 1861858263 393597842 818038768 772774169 466164234 2128340892 1087096208 1654010124 1164072500 377516627 1888675339 2079277227 2128873558 1447216044 1275850861 1945169720 1566826486 902844053 192379996 2034245143 1797636853 219176891 1288620340 467910549 2034840996 1073532624 1248622136 418723294 576330523 1391776852 1964446461 1177477458 1873712200
62
500731620 2036689606 2140735764 1246848288 270404716 1025108169 1506000691 968387319 1884229418 2045114167 227775361 2106508430 570356794 656712019 454678824 266854324 247401631 603332390 1448331694 1235939735 829685658 1561983417 1585079206 1595729931 672272379 1306723784 256821373 858592070 1368610511 1916726884 249245306 1515504643 1330489509 1366002344 916399426 568339425 669056646 1377209464 585338048 1594716799 1325280151 733503692 509707372 12470813 114619413 1409188154 1256549099 1434787983 259972614 850070929 1335284947 1995131352 1543185089 471467494 1064381178 1725210419 218126045 339131673 958058999 372666470 2039749087 889037192
62
474893297 1249195568 46008499 350966553 99101187 1145037800 600568051 1288579746 908599582 1664662881 1872527165 155701405 626596482 1172123739 1617617381 816565491 373968652 463352373 1026975445 1007253477 1907668757 950358163 332108188 1016491765 232327157 545516290 2135448234 56970105 356582728 1261847995 136207942 232238073 934841316 2120855100 711653013 310726440 1008291524 1828966297 1767255418 1554841748 1319254178 1103968107 165592380 1523705578 1185273548 1117096954 1086752071 835669037 1392431608 2112958051 1581706507 25538160 253415198 1368406213 448170592 756970870 1948681722 674507633 1590006829 1818445118 2039991052 1717344091
