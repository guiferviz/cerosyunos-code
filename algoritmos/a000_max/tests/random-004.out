2126270827
2131574079
2109667415
2111012224
2133157827
2141711101
1991291869
2130269096
2069610966
2042077592
2119494938
2004380453
2006014754
2041312901
2138091213
2109695018
2132788483
2138750975
2128377198
2072866278
2137654538
2092038462
2124719015
2077750259
2102919450
2128873558
2140735764
2135448234
